dev/side-nav.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Side nav</title>
+    <script type="module" src="./common.js"></script>
+    <script type="module">
+      import '@vaadin/icon';
+      import '@vaadin/icons';
+      import '@vaadin/scroller';
+      import "@vaadin/side-nav/enable";
```

Comment:
Please use `.js` extension for consistency with other dev pages. Same applies to `src` files.

Reply:
I checked the other dev pages, and most of them seem to follow this approach (no extension). If there is a decision to use the extension explicitly, I can do that.